Dockerfile.test-infra

```diff
@@ -14,12 +14,12 @@ RUN mkdir -p /root/.terraform.d/plugins
 RUN curl -SL https://github.com/dmacvicar/terraform-provider-libvirt/releases/download/v0.6.2/terraform-provider-libvirt-0.6.2+git.1585292411.8cbe9ad0.Fedora_28.x86_64.tar.gz | \
   tar -xz -C /root/.terraform.d/plugins
 
-# RUN curl -SL https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/4.6.0-rc.4/openshift-client-linux-4.6.0-rc.4.tar.gz | tar -xz -C /usr/bin
+RUN curl -SL https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/4.6.0/openshift-client-linux-4.6.0.tar.gz | tar -xz -C /usr/bin
```

Comment:
didn't it fail?

Reply:
They published a stable 4.6.0 version, not rc